lib/osf-components/addon/components/registries/schema-block-renderer/template.hbs

```diff
@@ -9,6 +9,7 @@
                 @uniqueID={{@uniqueID}}
                 @isRequired={{@isRequired}}
                 @isFieldsetGroup={{@isFieldsetGroup}}
+                @draftManager={{@draftManager}}
```

Comment:
This assumes that every rendering strategy needs `@draftManager` -- instead of passing it down separately, seems cleaner to pass `@draftManager` to the strategy for the review page. Does that seem reasonable?